react-components/src/architecture/base/commands/BaseCommand.ts

```diff
@@ -53,7 +57,11 @@ export abstract class BaseCommand {
   // =================================================
 
   public get name(): string {
-    return this.tooltip.fallback;
+    if (this.tooltip === undefined) {
+      return '';
+    }
+
+    return isTranslatedString(this.tooltip) ? this.tooltip.key : this.tooltip?.untranslated;
```

Comment:
Maybe this should be translated - now we will use the key (which is not meant to be human-readable), if it's a name that's supposed to be translated